config.json

```diff
@@ -365,29 +365,29 @@
   "concepts": [],
   "key_features": [
     {
-      "title": "Compile to JavaScript",
-      "content": "Allowing you to use CoffeeScript anywhere JavaScript is used.",
-      "icon": "web"
+      "title": "Transcompiles to JavaScript",
```

Comment:
I don't know about Transcompiles. I know compiles and transpiles, but not transcompiles. Is that a term I should be knowing?

Reply:
It is more [popularly known](https://www.computerhope.com/jargon/t/transcompiler.htm) as transpiling, it seems. A [tutorial](https://www.tutorialspoint.com/coffeescript/index.htm) I used utilized this term, but I think it's better if we change it to transpiles.

Reply:
As it is, the configlet failed because this title was too long. I changed it.